cmd/kubectl-gadget/dns.go

```diff
@@ -64,11 +64,12 @@ var dnsCmd = &cobra.Command{
 		}
 
 		config := &utils.TraceConfig{
-			GadgetName:       "dns",
-			Operation:        "start",
-			TraceOutputMode:  "Stream",
-			TraceOutputState: "Started",
-			CommonFlags:      &params,
+			GadgetName:        "dns",
+			Operation:         "start",
+			TraceOutputMode:   "Stream",
+			TraceOutputState:  "Started",
+			TraceInitialState: "Started",
```

Comment:
I think it is useful for the DNS gadget to use this new feature. It will wait twice for the same status in a single CLI call.